samples/Avalonia.Labs.Catalog/ViewModels/MainViewModel.cs

```diff
@@ -46,5 +49,18 @@ public async void NavigateTo(object page)
                 await NavigationRouter.NavigateToAsync(page);
             }
         }
+
+        public void SwapTheme(object parameter)
+        {
+            darkTheme = !darkTheme;
+            if (darkTheme)
+            {
+                App.Current!.RequestedThemeVariant = ThemeVariant.Dark;
+            }
+            else
+            {
+                App.Current!.RequestedThemeVariant = ThemeVariant.Default;
+            }
```

Comment:
If the user has Dark theme in the system, it will only switch between Dark and Dark.